Indicators/IndicatorExtensions.cs

```diff
@@ -61,6 +61,14 @@ public static bool Update(this IndicatorBase<IndicatorDataPoint> indicator, Date
             return second;
         }
 
+        public static SimpleMovingAverage SMA<T, TSecond>(this IndicatorBase<T> left, int period, bool waitForFirstToReady = true)
```

Comment:
Can you please add a XML comment here? It will help autocomplete users understand how to use it